internal/protos/options.pb.go

```diff
@@ -35,11 +35,11 @@ type FieldOptions struct {
 	Ignore bool `protobuf:"varint,1,opt,name=ignore,proto3" json:"ignore,omitempty"`
 	// Fields tagged with this will be marked as "required" in generated schemas
 	Required bool `protobuf:"varint,2,opt,name=required,proto3" json:"required,omitempty"`
-	// Fields tagged with this will be constrain strings using the "minLength" keyword in generated schemas
+	// Fields tagged with this will constrain strings using the "minLength" keyword in generated schemas
```

Comment:
I'm not sure why these weren't already re-generated. Good that you picked this up here!

Reply:
I must have just been lazy, not run `make generate` for a while.

Reply:
Yeah, initially I though about having separate commit for this, however in the end I figured it would not be too big of a change to just have it in same one.